statistics/histogram.go

```diff
@@ -352,7 +353,8 @@ func (hg *Histogram) RemoveVals(valCntPairs []TopNMeta) {
 			// If the current val smaller than current bucket's lower bound, skip it.
 			cmpResult = bytes.Compare(hg.Bounds.Column(0).GetRaw(bktIdx*2), valCntPairs[pairIdx].Encoded)
 			if cmpResult > 0 {
-				continue
+				pairIdx++
```

Comment:
Should we increase `pairIdx` also at line:362?

Reply:
For line 362, it's the case `bucket.upper < current pair`. We need to go to the next bucket. The pair doesn't need to move.

Reply:
No. If the current val bigger than the current bucket's upper bound, we need to increase `bktIdx`. So we only break here.